components/molecule/dropdownOption/src/_settings.scss

```
@@ -4,6 +4,9 @@ $mih-dropdown-option: 40px !default;
 $h-dropdown-option: $mih-dropdown-option !default; // deprecated
 $bgc-dropdown-option-hover: $c-gray-light-4 !default;
 $c-dropdown-option-hover: inherit !default;
+$oc-dropdown-option-focus-visible: $c-primary !default;
+$oo-dropdown-option-focus-visible: $m-xs !default;
+$ow-dropdown-option-focus-visible: 3px !default;
```

```suggestion
$ow-dropdown-option-focus-visible: 2px !default;
```

el default es 2 en el resto de elementos